browser.go

```diff
@@ -310,6 +310,15 @@ func (b *Browser) run(ctx context.Context) {
 	}
 }
 
+// Process returns the Chrome process object
+// Example:
+//     if process := chromedp.FromContext(ctx).Browser.Process(); process != nil {
+//         fmt.Printf("Chrome PID: %v", process.Pid)
+//     }
```

Comment:
This example is good.
You mentioned that "need a pid to control memory for all Chrome processes", it would be better if you can elaborate on that.

Reply:
Done.